test/test.js

```diff
@@ -50,24 +50,24 @@ More commands and details:
 		}).catch(e => done(e));
 	});
 
-	it("good evening", function(done) {
-		const helper = new Helper('../scripts/polite.js');
+	it("blahwithspace", function(done) {
+		const helper = new Helper('../scripts/search.js');
 		room = helper.createRoom();
 		const command = '@hubot ' + this.test.title;
 		co(function *() {
 			yield room.user.say('alice', command);
-			expect(room.messages.length).to.eql(2);
+			setTimeout(() => expect(room.messages.length).to.eql(2), 100);
 			done();
```

Comment:
This looks like a race condition, the `done()` can fire before the `setTimeout` above. Could we move the `done()` inside the `setTimeout`?